packages/analytics/shippers/elastic_v3/server/src/server_shipper.ts

```diff
@@ -216,40 +216,31 @@ export class ElasticV3ServerShipper implements IShipper {
   }
 
   private setInternalSubscriber() {
-    // Check the status of the queues every 1 second.
     merge(
-      interval(1000).pipe(takeUntil(this.shutdown$)),
-      // Using a promise because complete does not emit through the pipe.
-      from(firstValueFrom(this.shutdown$, { defaultValue: true }))
+      merge(
+        // Attempt to send every 10 seconds
+        interval(10 * SECOND).pipe(takeUntil(this.shutdown$)),
+        // React to opt in changes. The filter below stops if not opted-in.
+        this.isOptedIn$,
+        // React to connectivity changes. The filter below stops if not online.
+        this.firstTimeOffline$
+      ).pipe(filter(() => Date.now() - this.lastBatchSent >= 10 * SECOND)),
```

Comment:
I grouped them and limited their emissions by the lastBatchSent to avoid many emissions too close in time (after all we want a pace of maximum 1 request every 10 seconds

Reply:
I see that we no longer have the `shutdown$` obs in the `merge()`. That means, in case of shutdown, data collected since the last `interval()` will be lost, right? Is that acceptable?

Reply:
It is outside THAT merge. But that merge is nested inside another merge :)

Reply:
let me rewrite this bit so it's more obvious :)